hyfetch-git/hyfetch-git.pacscript

```diff
@@ -0,0 +1,22 @@
+pkgname="hyfetch-git"
+gives="hyfetch"
+pkgdesc="Neofetch with LGBTQ pride flags."
+url="https://github.com/hykilpikonna/hyfetch"
+source=("https://github.com/hykilpikonna/hyfetch.git")
+maintainer=("pythoncrazy <pinakpaliwal07@gmail.com>")
+pkgver="1.4.11"
+depends=('python3')
+makedepends=('python3-pip')
+external_connection="true"
+
+build() {
+  cd "${_archive}"
+  pip install .
+}
+
+package() {
+  cd "${_archive}"
```

Comment:
Do not use `${_archive}` in new scripts.

Reply:
What is the modern alternative? Is it `cd "${srcdir}/${pkgname%-git}"`? Thank you for all of your wonderful help and support!

Reply:
Well first of all, it should be `${pkgname%-git}`, but I would suggest just not making source download without the git, so then you wouldn't have to have `${pkgname%-git}` everywhere, and just have `${pkgname}`.